src/publicApiClient.ts

```diff
@@ -27,8 +30,10 @@ export class PublicApiClient extends AbstractRestfulClient {
    * Creates a new {@link CustomersClient} instance and returns it
    * @returns {@link CustomersClient}
    */
-  public getCustomersClient(): CustomersClient {
-    return new CustomersClient()
+  public getCustomersClient(
+    configuration?: ConfigurationsClient,
```

Comment:
To enable logging on a Client, pass `is_logging: true` in the client Constructor. By default, it is disabled.